src/styles/main.global.css

```diff
@@ -41,6 +41,14 @@
   .heading-4 {
     @apply text-xl font-semibold;
   }
+
+  .doc-list-title {
```

Comment:
I think these styles should be placed in the `.css` file in the component where they used.